journal.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+import os
+import datetime
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+
+
+# DATABASE_URL = os.environ.get(
+#     'DATABASE_URL',
+#     'postgresql://jameshemmaplardh:123@localhost:5432/learning-journal'
+# )
+
+
+
+Base = declarative_base()
+
```

Comment:
Too few blank lines here.  Again, your editor should be telling you this.  make sure you have code linting working properly